rubocop.rb

```diff
@@ -156,14 +156,17 @@ def in_diff?(changed_files, path, line)
 
   body = <<~BODY
     <!-- rubocop-comment-id: outside-diff -->
-    Rubocop offenses found outside of the diff:
-
+    <details>
+      <summary>Rubocop offenses found outside of the diff</summary>
+      <br>
```

Comment:
you mentioned it, but there should be an empty line instead of the `<br>`, like mc-atlantis does https://github.com/instructure/mc-terraform/pull/444#issuecomment-2428557812

Reply:
`<br>`
<details>
  <summary>TEST</summary>
  <br>
**lib/tasks/materials.rake:1**
[Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
</details>

--

`<br>` + `\n`
<details>
  <summary>TEST</summary>
  <br>

**lib/tasks/materials.rake:1**
[Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
</details>

--

`\n`
<details>
  <summary>TEST</summary>

**lib/tasks/materials.rake:1**
[Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
</details>